src/index.js

```diff
@@ -11,13 +11,25 @@ import * as watch from './watch.js';
 
 log.info('🗿 npm ↔️ Algolia replication starts ⛷ 🐌 🛰');
 
+const KILL_PROCESS_EVERY_MS = 12 * 60 * 60 * 1000; // every 12 hours
+
 /**
  * Main process
  *   - Bootstrap: will index the whole list of packages (if needed)
  *   - Watch    : will process update in real time
  */
 async function main() {
   const start = Date.now();
+
+  // We schedule to kill the process:
+  //  - reset cache
+  //  - maybe retrigger bootstrap
+  setInterval(() => {
```

Comment:
```suggestion
  setTimeout(() => {
```

Does this have an impact on perf, I've heard other times long-running timers can take up much more memory than expected, or will it be fine?

Reply:
I tried to find info about this, I think the way nodejs is managing the even loop is making it trivial (or at least consistant) wether it's long or not.

Reply:
NB: you can have up to 25days (32bit) of timeout, so they probably handle that fine